### PR TITLE
Support submit GPU jobs

### LINKED ISSUE #64: Paddle cloud support GPU mode
- GPU Paddle cloud job Docker image
- Trainer Job YAML support GPU configuration.

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix #64

### CODE REVIEW

File: paddlecloud/paddlecloud/settings.py [hunk 1]
```diff
@@ -295,10 +297,17 @@
 # where cephfs root is mounted when using cephfs storage service
 STORAGE_PATH="/pfs"
 
+# ===================== Docker image registry =====================
 JOB_DOCKER_IMAGE = {
-    "image": "yancey1989/paddlecloud-job",
-    "registry_secret": "job-registry-secret",
+    # These images are built by `docker/build_docker.sh` under this repo.
+    "image": "typhoon1986/paddlecloud-job",
+    "image-gpu": "typhoon1986/paddlecloud-job:gpu",
```
Comment (Yancey1989) [inline paddlecloud/paddlecloud/settings.py]: Use unerscores style: `image-gpu` => `image_gpu`

File: paddlecloud/paddlejob/views.py [hunk 2]
```diff
@@ -63,6 +63,25 @@ def post(self, request, format=None):
             pass
 
         registry_secret = settings.JOB_DOCKER_IMAGE.get("registry_secret", None)
+        # get user specified image
+        job_image = obj.get("image", None)
+        gpu_count = obj.get("gpu", 0)
+        # use default images
+        if not job_image :
+            if gpu_count > 0:
+                job_image = settings.JOB_DOCKER_IMAGE["image-gpu"]
```
Comment (Yancey1989) [inline paddlecloud/paddlejob/views.py]: `image-gpu` => `image_gpu`

File: paddlecloud/paddlecloud/settings.py (hunk 1 above)
Comment (typhoonzero) [inline paddlecloud/paddlecloud/settings.py]: Done

File: paddlecloud/paddlejob/views.py (hunk 2 above)
Comment (typhoonzero) [inline paddlecloud/paddlejob/views.py]: Done

**Pull request merged**